### PR TITLE
added playerMinter

### PR DESCRIPTION
**Pull request by MiyaSteven**

### CODE REVIEW

File: LavaFlow.cdc [hunk 1]
```diff
@@ -198,6 +199,23 @@ pub contract LavaFlow {
         }
     }
 
+    pub resource PlayerMinter {
```
Comment (dangdennis) [inline LavaFlow.cdc:202]: Can you add a comment?

**Review by dangdennis — APPROVED** [review]
Nicesuhh

File: LavaFlow.cdc (hunk 1 above)
Comment (MiyaSteven) [reply LavaFlow.cdc:202]: What should go in a comment?

Comment (dangdennis) [reply LavaFlow.cdc:202]: You can look at the other comments. Intent and light description is usually enough.

I follow Go’s style where everything that is a public API gets a comment.

**Pull request merged**